payments/models.py

```diff
@@ -276,9 +277,12 @@ def handle_post_sale(self):
     def send_ecommerce_order_receipt(self):
         """
         Send the receipt email.
-
-        TODO: add email
         """
+        successful_order_payment_email_task.delay(
```

Comment:
Can this be done in a hook? There's already a hook for this (`PostSaleSendEmails` in `payments/hooks/post_sale.py` - it's a stub) so just lifting this call and placing it there would take care of it. (Small note: there's some changes pending in [this PR](https://github.com/mitodl/unified-ecommerce/pull/157) that fixes some issues with event firing that may affect testing.)